src/class_singleCleaner.py

```diff
@@ -48,20 +48,15 @@ class singleCleaner(StoppableThread):
     def run(self):  # pylint: disable=too-many-branches
         gc.disable()
         timeWeLastClearedInventoryAndPubkeysTables = 0
-        try:
-            state.maximumLengthOfTimeToBotherResendingMessages = (
-                float(BMConfigParser().get(
```

Comment:
move the float (instead of casting, do `BMConfigParser().getfloat()`

Reply:
Updated